internal/pkg/deploy/cloudformation/stack/env.go

```diff
@@ -423,11 +424,18 @@ func (e *EnvStackConfig) vpcConfig() (template.VPCConfig, error) {
 	return template.VPCConfig{
 		Imported:            e.importVPC(),
 		Managed:             e.managedVPC(),
-		AllowVPCIngress:     aws.BoolValue(e.in.Mft.HTTPConfig.Private.SecurityGroupsConfig.Ingress.VPCIngress),
+		AllowVPCIngress:     e.isAllowVPCIngress(),
 		SecurityGroupConfig: securityGroupConfig,
 	}, nil
 }
 
+func (e *EnvStackConfig) isAllowVPCIngress() *bool {
```

Comment:
can this be instead a method in the `manifest` package similar to `IsPublicLBIngressRestrictedToCDN`:
`IsPrivateLBIngressAllowingVPC` ?